**Exercise 3.30** A 4-31G basis for He has not been officially defined. Huzinaga, however, in an SCF calculation on the He atom using four uncontracted \( 1s \) Gaussians, found the coefficients and optimum exponents of the normalized \( 1s \) orbital of He to be:

| $\alpha_\mu$ | $C_\mu$ |
|--------------|---------|
| 0.298073     | 0.51380 |
| 1.242567     | 0.46954 |
| 5.782948     | 0.15457 |
| 38.47497     | 0.02373 |

Use the expression for overlaps given in Appendix A to derive the contraction parameters for a 4-31G He basis set.


# The 4-31G basis is a split valence basis.
The valence shell orbitals are each split into two Gaussian basis functions.
Here the 1s orbial is in the valence shell as the only orbital.
One basis function is more diffuse, called the outer.
The other basis function is tighter, called the inner.

In 4-31G 4 Gaussian primitives are contracted for each core orbital basis function; 3 Guassian primitives are contracted for a inner valence basis function; 1 Gaussian primitive is left uncontracted for an outer valence basis function.

The outer basis function is the most diffuse of the primitive Gaussians, so in this case it is:

$\phi''_{1s}(r) = g_{1s}(0.298073, r)$


# Inner basis contraction

$\phi'_{1s}(r) = N \big[ 0.46954\, g_{1s}(1.242567, r) + 0.15457\, g_{1s}(5.782948, r) + 0.02373\, g_{1s}(38.47497, r) \big]$

# Overlap integrals
The question points to Appendix A for the overlap integrals.
This unhelpfully gives overlaps for unnormalised Gaussians.
Turning to page 153 we see that the equation 3.203 gives the normalisation constant for Guassian-type primitives is

$(\frac{2\alpha}{\pi})^{\frac{3}{4}}$ .

Appendix A equation A.9 gives the overlap integral between two unnormalised Gaussians with a common centre to be

$(\frac{\pi}{\alpha + \beta})^\frac{3}{2}$

where $\alpha$ and $\beta$ are the Gaussian exponents.
Therefore, the overlap integral between two normalised centred Gaussians is

$(\frac{2\sqrt{\alpha\beta}}{\alpha+\beta})^{\frac{3}{2}}$

We can now code the modulus squared of $\phi'_{1s}(r)$ to find the normalisation factor N.

In [13]:
from math import sqrt

alpha = [1.242567, 5.782948, 38.47497]
coeff = [0.46954, 0.15457, 0.02373]
primitives = list(zip(alpha, coeff))

mod_sq = 0
for term1 in primitives:
    for term2 in primitives:
        alpha = term1 [0]
        beta = term2 [0]
        c1 = term1 [1]
        c2 = term2 [1]
        mod_sq += c1 * c2 * (2 * sqrt(alpha * beta) / (alpha + beta)) ** (3/2)

N = 1 / sqrt(mod_sq)
print(N)
print(N*coeff[0], N*coeff[1], N*coeff[2])

1.6895281164968035
0.7933010318199091 0.2611503609669109 0.040092502204469145


# Inner basis function
This value for N=1.695 gives
$\phi'_{1s}(r) = 0.79330\, g_{1s}(1.242567, r) + 0.26115\, g_{1s}(5.782948, r) + 0.04009\, g_{1s}(38.47497, r)$

This reproduces the answers provided on GitHub.